xml/System.ServiceModel.Channels/ServiceChannelProxy.xml

```diff
@@ -1,1536 +1,1540 @@
+      <MemberSignature Language="VB.NET" Value="Custom Event UnknownMessageReceived As EventHandler(Of UnknownMessageReceivedEventArgs) Implements System.ServiceModel.IClientChannel" />
+      <MemberSignature Language="F#" Value="member this.System.ServiceModel.IClientChannel.UnknownMessageReceived : EventHandler&lt;System.ServiceModel.UnknownMessageReceivedEventArgs&gt; " Usage="member this.System.ServiceModel.IClientChannel.UnknownMessageReceived : System.EventHandler&lt;System.ServiceModel.UnknownMessageReceivedEventArgs&gt; " />
+      <MemberType>Event</MemberType>
+      <Implements>
+        <InterfaceMember>E:System.ServiceModel.IClientChannel.UnknownMessageReceived</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.EventHandler&lt;System.ServiceModel.UnknownMessageReceivedEventArgs&gt;</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>This is a reserved event.</summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="System.ServiceModel.IClientChannel.Via">
+      <MemberSignature Language="C#" Value="Uri System.ServiceModel.IClientChannel.Via { get; }" />
+      <MemberSignature Language="ILAsm" Value=".property instance class System.Uri System.ServiceModel.IClientChannel.Via" />
+      <MemberSignature Language="DocId" Value="P:System.ServiceModel.Channels.ServiceChannelProxy.System#ServiceModel#IClientChannel#Via" />
+      <MemberSignature Language="VB.NET" Value=" ReadOnly Property Via As Uri Implements IClientChannel.Via" />
+      <MemberSignature Language="F#" Value="member this.System.ServiceModel.IClientChannel.Via : Uri" Usage="System.ServiceModel.IClientChannel.Via" />
+      <MemberSignature Language="C++ CLI" Value="property Uri ^ System::ServiceModel::IClientChannel::Via { Uri ^ get(); };" />
+      <MemberType>Property</MemberType>
+      <Implements>
+        <InterfaceMember>P:System.ServiceModel.IClientChannel.Via</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Uri</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Gets the URI that contains the transport address to which messages are sent on the client channel.</summary>
+        <value>The <see cref="T:System.Uri" /> that contains the transport address to which messages are sent on the channel.</value>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Channels.ServiceChannelProxy&gt; instance is cast to an &lt;xref:System.ServiceModel.IClientChannel&gt; interface.</remarks>

```

Comment:
```suggestion
        <remarks>This member is an explicit interface member implementation. It can be used only when the <see cref="T:System.ServiceModel.Channels.ServiceChannelProxy"/> instance is cast to an <see cref="T:System.ServiceModel.IClientChannel"/> interface.</remarks>

```